typescript/utils/src/addresses.test.ts

```diff
@@ -16,6 +16,11 @@ const COS_NON_ZERO_ADDR =
 const SOL_ZERO_ADDR = '111111';
 const SOL_NON_ZERO_ADDR = 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb';
 
+const FUEL_ZERO_ADDR =
+  '0x0000000000000000000000000000000000000000000000000000000000000000';
+const FUEL_NON_ZERO_ADDR =
+  '0x0000000000000000000000000000000000000000000000000000000000000001';
```

Comment:
A valid address makes it a bit nicer
```suggestion
  '0xCCae08F2eB25ce643496b0d0857F35DCfD0b30d5841EA10a078b79438C32AE07';
```